src/main/java/io/jenkins/plugins/coverage/CoverageAction.java

```diff
@@ -1,25 +1,27 @@
 package io.jenkins.plugins.coverage;
 
-import hudson.model.Action;
-import hudson.model.HealthReport;
-import hudson.model.HealthReportingAction;
-import hudson.model.Run;
+import hudson.model.*;
```

Comment:
Better to avoid that in the production code. It may lead to import ambiguity in some edge cases